### ISSUE TITLE
Incorrect output

### ISSUE THREAD

**Issue by FatherMcGruder** [cmayes/ofxstatement-betterment#2]
I tried converting the transactions.csv file I downloaded from Betterment, but when I attempt to import into GnuCash, I just see a bunch of zero value transactions. I see transactions for each of the market changes, dividends, and deposits, but the values are all zero. I should mention that before I attempted the conversion, I noticed that the config.ini file was non-existent, so I made one and copied in the sample text that you had in the readme. If you think that that could be the problem, could you maybe provide a config.ini file that you know works?

**Comment by cmayes** [comment]
Yeah, that's pretty bizarre.  I've never used GnuCash before, so I installed it to see whether I could reproduce the issue.  Indeed, I can.

GnuCash 2.6.12 seems to recognize just about every field but the TRNAMT field: I was even able to manually add a NAME field and GnuCash picked it up.

I didn't see any obvious answers in the GnuCash docs or on Google as to why GnuCash would refuse to read the transaction value.  I've tried changing the transaction type to CREDIT or DEBIT, made the transaction ID numeric, and a few other things, but the amount is still zero.

I'll see whether I can figure this out, but I'm not sure what the issue is.  I don't see any errors when I make the import, and the file works fine in Moneydance.  Please do let me know if you see anything that might indicate what the issue is.

**Comment by FatherMcGruder** [comment]
I am not familiar with Moneydance, but I quickly perused their site and did not see a mention of double entry accounting. Could that have something to do with it?

Kurt

On June 13, 2016 9:36:08 PM EDT, Chris Mayes notifications@github.com wrote:

> Yeah, that's pretty bizarre.  I've never used GnuCash before, so I
> installed it to see whether I could reproduce the issue.  Indeed, I
> can.
> 
> GnuCash 2.6.12 seems to recognize just about every field but the TRNAMT
> field: I was even able to manually add a NAME field and GnuCash picked
> it up.
> 
> I didn't see any obvious answers in the GnuCash docs or on Google as to
> why GnuCash would refuse to read the transaction value.  I've tried
> changing the transaction type to CREDIT or DEBIT, made the transaction
> ID numeric, and a few other things, but the amount is still zero.
> 
> I'll see whether I can figure this out, but I'm not sure what the issue
> is.  I don't see any errors when I make the import, and the file works
> fine in Moneydance.  Please do let me know if you see anything that
> might indicate what the issue is.
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/cmayes/ofxstatement-betterment/issues/2#issuecomment-225757099

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by FatherMcGruder** [comment]
In other words, I think that GnuCash is expecting two accounts for each transaction. If ofxstatement is only giving it one account per transaction, the transactions might have zero value since it appears to GnuCash that funds aren't going anywhere.

Kurt

On June 13, 2016 11:02:14 PM EDT, Kurt Padilla klp@riseup.net wrote:

> I am not familiar with Moneydance, but I quickly perused their site and
> did not see a mention of double entry accounting. Could that have
> something to do with it?
> 
> Kurt
> 
> On June 13, 2016 9:36:08 PM EDT, Chris Mayes notifications@github.com
> wrote:
> 
> > Yeah, that's pretty bizarre.  I've never used GnuCash before, so I
> > installed it to see whether I could reproduce the issue.  Indeed, I
> > can.
> > 
> > GnuCash 2.6.12 seems to recognize just about every field but the
> > TRNAMT
> > field: I was even able to manually add a NAME field and GnuCash picked
> > it up.
> > 
> > I didn't see any obvious answers in the GnuCash docs or on Google as
> > to
> > why GnuCash would refuse to read the transaction value.  I've tried
> > changing the transaction type to CREDIT or DEBIT, made the transaction
> > ID numeric, and a few other things, but the amount is still zero.
> > 
> > I'll see whether I can figure this out, but I'm not sure what the
> > issue
> > is.  I don't see any errors when I make the import, and the file works
> > fine in Moneydance.  Please do let me know if you see anything that
> > might indicate what the issue is.
> > 
> > ---
> > 
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub:
> > https://github.com/cmayes/ofxstatement-betterment/issues/2#issuecomment-225757099
> 
> ## 
> 
> Sent from my Android device with K-9 Mail. Please excuse my brevity.

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.
